contracts/test/TestMathUtils.sol

```diff
@@ -17,12 +17,17 @@ contract TestMathUtils {
     }
 
     function test_percOf2() public {
-        Assert.equal(MathUtils.percOf(100, 3), 0, ".0003% of 100 is 0");
-        Assert.equal(MathUtils.percOf(100, 100000), 10, "10% of 100 is 10");
+        Assert.equal(MathUtils.percOf(100, 3), 0, ".0000000000000000000000003% of 100 is 0");
+        Assert.equal(MathUtils.percOf(10**27, 1), 1, ".0000000000000000000000001% of 1000000000 is 1");
+        Assert.equal(MathUtils.percOf(100, 10**27 / 10), 10, "10% of 100 is 10");
     }
 
     function test_percPoints() public {
-        Assert.equal(MathUtils.percPoints(3, 4), 750000, "3/4 should convert to valid percentage");
-        Assert.equal(MathUtils.percPoints(100, 300), 333333, "100/300 should convert to valid percentage");
+        Assert.equal(MathUtils.percPoints(3, 4), 750000000000000000000000000, "3/4 should convert to valid percentage");
```

Comment:
scary numbers

Reply:
Open to suggestions, but feels out of scope for this PR